my_compassion/static/src/css/my2_login.css

```diff
@@ -0,0 +1,174 @@
+/*
+    This stylesheet concerns the my2_login_template component
+    templates/my2_login_template.xml
+*/
+
+/* Header container styling */
+.header-elements {
+  position: relative;
+  width: 100%;
+  padding: 1rem;
+  margin-bottom: 20rem;
+}
```

Comment:
This seems related to the menu, it should be place else where then my2_login.css and be part of another PR
```suggestion
```